sandec/get_dll.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
```

Comment:
Using `set -euo pipefail` makes bash development a little easier and more robust.

Reply:
That's nice, didn't know about that either